<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Usermenu</class>
 <widget class="QDialog" name="Usermenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1246</width>
    <height>604</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableView" name="SCtable">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>50</y>
     <width>661</width>
     <height>351</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Button1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>460</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>查询商品</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button0">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>530</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>注销登录</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>460</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>搜索商品（按名称）</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>530</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>查看购物车</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button3">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>460</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>搜索商品（按品牌）</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button4">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>460</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>模糊搜索</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button6">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>530</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>添加商品至购物车</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button7">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>530</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>删除购物车中商品</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button8">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>530</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>结账</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>541</width>
     <height>401</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>商品</string>
    </attribute>
    <widget class="QTableView" name="Itemtable">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>501</width>
       <height>351</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>历史</string>
    </attribute>
    <widget class="QTableView" name="Historytable">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>501</width>
       <height>351</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="Button9">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>460</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>查询历史购买</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button10">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>530</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>修改密码</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>20</y>
     <width>54</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>购物车</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>410</y>
     <width>371</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
